frontend/src/employee-frontend/api/applications.ts

```diff
@@ -411,6 +411,7 @@ export async function getClubTerms(): Promise<Result<ClubTerm[]>> {
   }
 }
 
+// TODO: refactor below function calls to leverage getPreschoolTermsV2()
```

Comment:
Maybe this could be renamed as e.g. `getPreschoolTermsResult` so the new one wouldn't need that `V2` suffix, which sounds more like calling a different API endpoint altogether. This could also internally use the other function so there's less duplication:
```
export async function getPreschoolTermsResult(): Promise<Result<PreschoolTerm[]>> {
  return getPreschoolTerms()
    .then(result => Success.of(result))
    .catch(e => Failure.fromError(e))
}
```

Reply:
Very good point, I will make this change.